src/acceptance/broker/broker_test.go

```diff
@@ -177,18 +177,14 @@ var _ = Describe("AutoScaler Service Broker", func() {
 		service.delete()
 	})
 
-	It("should fail to update service instance from acceptance-standard to first", func() {
+	It("should fail to update a service instance from the second plan to the first plan", func() {
 		plans := getPlans()
 		if plans.length() < 2 {
 			Skip(fmt.Sprintf("2 plans needed, only one plan available plans:%+v", plans))
 			return
 		}
-		if !plans.contains("acceptance-standard") {
```

Comment:
I forgot it, why are we going to make this independant of the plan names?

Reply:
Mainly, so that it also runs on our deployments at SAP.

But now that I think about it, the correct fix would be to check if the plan is updateable and only then decide if to skip or not.

But in the end what would we be testing here? The Cloud Controller, as it already checks if a plan is marked as `plan_updateable`.

So, I think I'm going to remove this acceptance test as this does not test **our** functionality.

Fortunately, our functionality is covered by https://github.com/cloudfoundry/app-autoscaler-release/blob/main/src/autoscaler/api/brokerserver/broker_handler_test.go#L583-L646

Reply:
This whole "can't update from first to second plan"-logic is completely misterious. 

I am always asking myself why we don't test that we can't update from plan a (updatable) to plan b (not updatable). 🤷

Reply:
Implemented in 64e23b9d9c6665baa9a0a27c407ca9691276d84c